src/struct_ls/point_relax.c

```diff
@@ -1026,41 +1029,40 @@ hypre_PointRelax_core12( void               *relax_vdata,
       {
          case 7:
             Ap6 = hypre_StructMatrixBoxData(A, boxarray_id, ssi[6]);
-            xoff6 = hypre_BoxOffsetDistance(
-                       x_data_box, stencil_shape[ssi[6]]);
+            xoff6 = hypre_BoxOffsetDistance(x_data_box, stencil_shape[ssi[6]]);
+            /* fall through */
```

Comment:
Good comment to add!

Reply:
BTW, I added this comment to silence this warning. See https://lwn.net/Articles/794944/

Reply:
Cool!  This was intentional, as you know, but the comment makes that explicit.